src/controllers/moneycontroller.js

```diff
@@ -11,7 +11,10 @@ export default class MoneyController extends Controller
      */
     static updateMoney(amount)
     {
-        GAME.model.config.money = GAME.model.config.money + parseFloat(amount);
-        $("#money").html(GAME.model.config.money.toFixed(2));
+        const config = GAME.model.config;
```

Comment:
I changed this because first I also updated score here (which still might be handy). Therefore it was useful to use the cont config, now it isn't necessary per se but forgot to change it back.